rkt/rkt.go

```diff
@@ -53,11 +53,11 @@ func init() {
 }
 
 type Command struct {
-	Name        string       // Name of the Command and the string to use to invoke it
-	Summary     string       // One-sentence summary of what the Command does
-	Usage       string       // Usage options/arguments
-	Description string       // Detailed description of command
-	Flags       flag.FlagSet // Set of flags associated with this command
+	Name        string        // Name of the Command and the string to use to invoke it
+	Summary     string        // One-sentence summary of what the Command does
+	Usage       string        // Usage options/arguments
+	Description string        // Detailed description of command
+	Flags       *flag.FlagSet // Set of flags associated with this command
```

Comment:
ah, missed that. I remember tackling this aaaaaaages ago with babby's first CLI library, can't remember if I ever came up with a decent solution though, will ponder. (maybe `Run func(fs *flags.FlagSet, args []string) int` or something)